packages/remix-dev/server-build.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable no-unreachable */
```

Comment:
This rule is explicitly turned off in typescript-eslint configs because it's covered by TypeScript.